layers/stateless/sl_buffer.cpp

```diff
@@ -226,4 +227,49 @@ bool Device::ValidateCreateBufferBufferDeviceAddress(const VkBufferCreateInfo &c
     return skip;
 }
 
+bool Device::ValidateCreateBufferTileMemory(const VkBufferCreateInfo &create_info, const Location &create_info_loc) const {
+    const VkBufferCreateFlags flags = create_info.flags;
+    const VkBufferUsageFlags2 usage = create_info.usage;
+    bool skip = false;
+
+    if (usage & VK_BUFFER_USAGE_TILE_MEMORY_BIT_QCOM) {
+        const VkBufferCreateFlags invalidFlagMask =
```

Comment:
nit:

```suggestion
        const VkBufferCreateFlags invalid_flag_mask =
```

and `valid_usage_mask`

and `invalid_flags`/`invalid_usage`